### ISSUE TITLE
Issue with neg_mean_squared_log_error metric and GradientBoostingRegressor

### ISSUE THREAD

**Issue by skatedplrn** [deeplearninc/auger-ml#142]
Run test_algorithm_regression2 and see:
ERROR:root:Score neg_mean_squared_log_error for algorithm sklearn.ensemble.GradientBoostingRegressor failed to build: Input contains NaN, infinity or a value too large for dtype('float64').

Needs to investigate

**Comment by skatedplrn** [comment]
Similar:
ERROR:root:Score neg_log_loss for algorithm auger_ml.algorithms.linear_svc.LinearSVC failed to build: Input contains NaN, infinity or a value too large for dtype('float64').

test_algorithm_binary_classification_LSVC

**Comment by skatedplrn** [comment]
test_algorithm_regression2 still report errors in the log
